cmd/crank/main.go

```diff
@@ -57,11 +61,14 @@ type cli struct {
 
 	// Flags.
 	Verbose verboseFlag `help:"Print verbose logging statements." name:"verbose"`
+
+	// Completion
+	Completions kongplete.InstallCompletions `cmd:"" help:"install shell completions"`
```

Comment:
do you think there's a slight bit more help text/hints we can give to the user so they have more idea about what to do with this command to get it working? i think we should also have our docs updated to be more clear and explicit, but in this short help output, could we also give a bit more help?

Reply:
Good Idea. I've also created a draft PR to the docs as it is tightly related:
https://github.com/crossplane/docs/pull/928